src/components/member-modal/MemberModal.styled.ts

```diff
@@ -0,0 +1,130 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  @media screen and (${({ theme }) => theme.desktop}) {
+    background: rgba(75, 84, 100, 0.8);
+    height: 100%;
+    left: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 80;
```

Comment:
On my laptop screen, the modal is too big for the window. Don't think it is worth changing now, but keep in mind to check the styling on multiple screen sizes.